evap/evaluation/migrations/0085_auto_20181015_1908.py

```diff
@@ -0,0 +1,28 @@
+# Generated by Django 2.0.9 on 2018-10-15 19:08
+
+from django.db import migrations, models
```

Comment:
can you give the migration a more descriptive filename?

Reply:
sure!